File: Problem Solving/C++/plus_minus.cpp
#include<iostream>
using namespace std;
int main()
{
    int n, i, plus, minus, zero;
    plus = minus = zero = 0;
    cin >> n;
    int arr[n];
    for(int i=0; i<n; i++)
    {
        cin >> arr[i];
        if(arr[i] < 0)
            minus++;
        else if(arr[i] > 0)
            plus++;
        else 
            zero++;
    }
    printf("%.6f\n%.6f\n%.6f", (float)plus/n, (float)minus/n, (float)zero/n);
    return 0;
}

File: Problem Solving/Python/plus_minus.py
if __name__ == '__main__':
    n = int(input())
    arr = list(map(int, input().split()))
    plus = minus = zero = 0
    for i in arr:
        if i < 0:
            minus += 1
        elif i > 0:
            plus += 1
        else:
            zero += 1

    print('{:.6f}\n{:.6f}\n{:.6f}'.format((plus/n), (minus/n), (zero/n)))